### PR TITLE
Improve CI so PR's build. Fix build.

### PR DESCRIPTION
**Pull request by krishnaglick**
## Change description

Improve CI so PR's build. Fix build.

### Development

- [x] Application changes have been tested appropriately

### Impact

- [x] Code follows company security practices and guidelines
- [x] Security impact of change has been considered
- [x] Performance impact of change has been considered
- [x] Possible migration needs considered (model migrations, config migrations, etc.)

Please explain any security, performance, migration, or other impacts if relevant:

### Code review

- [x] Pull request has a descriptive title and context useful to a reviewer. Screenshots or screencasts are attached where applicable.
- [x] Relevant tags have been added to the PR (bug, enhancement, internal, etc.)

### CODE REVIEW

File: core/src/classes/commonModel.ts [hunk 1]
```diff
@@ -13,8 +13,13 @@ import validator from "validator";
...
 import { Op, Attributes, where, fn, col } from "sequelize";
 import { config } from "actionhero";
-import { WhereOptions } from "sequelize/types";
+import { WhereOptions, WhereOperators } from "sequelize/types";
 import { Errors } from "../modules/errors";
+import { Where } from "sequelize/types/utils";
+
+type PartialRecord<K extends keyof any, T> = {
+  [P in K]?: T;
+};
```
Comment (krishnaglick) [inline core/src/classes/commonModel.ts]: Could put this elsewhere if we wanted.

File: tools/merger/lockfile [hunk 2]
```diff
@@ -4,8 +4,5 @@ set -e
 cd "$(dirname "$0")"
 cd ../..
 
-BRANCH=`git rev-parse --abbrev-ref HEAD`
-if [[ $BRANCH == 'main' ]]; then
```
Comment (edmundito) [inline tools/merger/lockfile:8]: How come this was removed?

File: core/src/classes/commonModel.ts (hunk 1 above)
Comment (edmundito) [reply core/src/classes/commonModel.ts]: Maybe `utils/types.ts`?

File: tools/merger/lockfile (hunk 2 above)
Comment (krishnaglick) [reply tools/merger/lockfile:8]: I _think_ it's the right piece to remove to ensure that CI always builds PR branches. But I may be wrong.

Comment (krishnaglick) [reply tools/merger/lockfile:8]: I think it was the right piece.

**Review by teallarson — APPROVED** [review]

**Pull request merged**